File: ProjektyZadania/Zadanie1/kalkulator.py
a = float(input("Podaj pierwszą liczbę: "))
b = float(input("Podaj drugą liczbę: "))
operator = input("Wybierz operację (+, -, *, /): ")

if operator == "+":
    wynik = a + b
elif operator == "-":
    wynik = a - b
elif operator == "*":
    wynik = a * b
elif operator == "/":
    if b != 0:
        wynik = a / b
    else:
        wynik = "Błąd,dzielenie przez zero"
else:
    wynik = "Nieznana operacja"

print("Wynik:", wynik)

File: ProjektyZadania/Zadanie1/kalkulator.cs
using System;

class Kalkulator {
    static void Main() {
        Console.Write("Podaj pierwszą liczbę: ");
        double a = Convert.ToDouble(Console.ReadLine());
        Console.Write("Podaj drugą liczbę: ");
        double b = Convert.ToDouble(Console.ReadLine());
        Console.Write("Wybierz operację (+, -, *, /): ");
        string op = Console.ReadLine();
        double wynik;
        if (op == "+")
            wynik = a + b;
        else if (op == "-")
            wynik = a - b;
        else if (op == "*")
            wynik = a * b;
        else if (op == "/")
            wynik = b != 0 ? a / b : double.NaN;
        else {
            Console.WriteLine("Nieznana operacja.");
            return;
        }
        Console.WriteLine("Wynik: " + wynik);
    }
}
